README.md

```diff
@@ -2,4 +2,119 @@
+| `auth.username`             | Sap Hana user                                                                                                                                                                                   | Required for Basic type.                  | hbadmin                                        |
+| `auth.password`             | Sap Hana password                                                                                                                                                                               | Required for Basic type.                  | pass                                           |
+| `auth.token`                | JWT token                                                                                                                                                                                       | Required for JWT type.                    | jwt_token                                      |
+| `auth.clientCertFilePath`   | Path for certification file                                                                                                                                                                     | Required for X509 type.                   | /tmp/file.cert                                 |
+| `auth.ClientKeyFilePath`    | Path for key file                                                                                                                                                                               | Required for X509 type.                   | /tmp/key.cert                                  |
+
+### Snapshot
+By default when the connector starts for the first time, snapshot mode is enabled, which means that existing data will
+be read. To skip reading existing, change config parameter `snapshot` to `false`.
+
+First time when the snapshot iterator starts work,
+it is get max value from `orderingColumn` and saves this value to position.
+The snapshot iterator reads all rows, where `orderingColumn` values less or equal maxValue, from the table in batches.
+
+
+Values in the ordering column must be unique and suitable for sorting, otherwise, the snapshot won't work correctly.
+Iterators saves last processed value from `orderingColumn` column to position to field `SnapshotLastProcessedVal`.
+If snapshot was interrupted on next start connector will parse last recorded position
+to find next snapshot rows.
+
+
+When all records are returned, the connector switches to the CDC iterator.
+
+### Change Data Capture (CDC)
+
+This connector implements CDC features for DB2 by adding a tracking table and triggers to populate it. The tracking
+table has the same name as a target table with the prefix `CONDUIT_` and suffix from time when pipeline started on
+format "hhmmss". For example for table `PRODUCTS` tracking will be looks like `CONDUIT_PRODUCTS_213315`. 
```

Comment:
"tracking will be looks like" -> "the tracking table's name will be"

Reply:
thanks, fixed